WeakAuras/WeakAuras.lua

```diff
@@ -4277,7 +4321,7 @@ end
 
 function WeakAuras.Add(data, takeSnapshot)
   if takeSnapshot then
-    WeakAuras.SetMigrationSnapshot(data.uid, CopyTable(data))
+    -- WeakAuras.SetMigrationSnapshot(data.uid, CopyTable(data))
```

Comment:
Oh, yes it is. When debugging something I needed to turn the snapshots off because they were breaking things. And then because I'm a genius, I didn't bother turning them back on.